_posts/2020-04-10-send-a-concert-stay-connected.markdown

```diff
@@ -0,0 +1,51 @@
+---
+title: Send a Concert, Stay Connected
+date: 2020-04-10 10:29:00 -04:00
+categories:
+- stories
+tags:
+- Curbside Concerts
+author: Veronica Johnson
+layout: post
+---
+
+<iframe width="1120" height="630" src="https://www.youtube.com/embed/UsMiRSII19g" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
this embed code goes over our right border on the page.
![Screen Shot 2020-04-10 at 11 57 04 AM](https://user-images.githubusercontent.com/221065/79004339-8f35e580-7b22-11ea-8fd2-69cfb179075a.png)

Reply:
fixed it manually, set width to 100% of it's container.